src/ga4gh-tes.sln

```diff
@@ -0,0 +1,49 @@
+﻿

```

Comment:
There already is a solution file in this repo, can we remove this?

Reply:
Yes, removed in new commit.